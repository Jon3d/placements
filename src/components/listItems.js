import * as React from 'react';
import ListItemButton from '@mui/material/ListItemButton';
import ListItemIcon from '@mui/material/ListItemIcon';
import ListItemText from '@mui/material/ListItemText';
import ListSubheader from '@mui/material/ListSubheader';
import BarChartIcon from '@mui/icons-material/BarChart';
import LayersIcon from '@mui/icons-material/Layers';

export const mainListItems = (
  <React.Fragment>
    <ListItemButton href="/campaigns">
      <ListItemIcon>
        <BarChartIcon />
      </ListItemIcon>
      <ListItemText primary="Campaigns" />
    </ListItemButton>
    <ListItemButton href="/details">
      <ListItemIcon>
        <LayersIcon />
      </ListItemIcon>
      <ListItemText primary="Details" />
    </ListItemButton>
  </React.Fragment>
);

export const secondaryListItems = (
  <React.Fragment>
    <ListSubheader component="div" inset>
      Saved reports
    </ListSubheader>
  </React.Fragment>
);