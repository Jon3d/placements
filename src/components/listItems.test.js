import renderer from 'react-test-renderer';
import { mainListItem, secondaryListItem } from './listItems';

it('renders mainListItem correctly', () => {
  const tree = renderer
    .create(<mainListItem />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders secondaryListItem correctly', () => {
  const tree = renderer
    .create(<secondaryListItem />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});