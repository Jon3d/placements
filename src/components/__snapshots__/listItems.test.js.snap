// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders mainListItem correctly 1`] = `<mainListItem />`;

exports[`renders secondaryListItem correctly 1`] = `<secondaryListItem />`;
